Jenkinsfile

```diff
@@ -70,103 +55,189 @@ pipeline {
+                  Map buildInfo = [
+                    "project_id": service_server,
+                    "build_id": env.BUILD_ID,
+                    "release_id": params.RELEASE_NAME,
+                    "git_tag": env.GIT_TAG
+                  ]
+                  createBuild(
+                    buildInfo,
+                    rmsSupportedEnvironment.staging //production
+                  )
+                }
+              }
+            }
+          }
+        }
+        stage('Publish - Secondary') {
+          steps {
+            script {
+              lastStage = env.STAGE_NAME
+              if (params.RELEASE_BUILD) {
+                env.BUILD_ID = dockerize_secondary.publish(sourceTag: env.SECONDARY_DOCKER_TAG)
+              } else {
+                env.BUILD_ID = dockerize_secondary.publish(
+                  sourceTag: env.SECONDARY_DOCKER_TAG,
+                  environments: ['internal']
+                )
+              }
+            }
+          }
+          post {
+            success {
+              script {
+                if (params.RELEASE_BUILD) {
+                  Map buildInfo = [
+                    "project_id": service_secondary,
+                    "build_id": env.SECONDARY_BUILD_ID,
+                    "release_id": params.RELEASE_NAME,
+                    "git_tag": env.GIT_TAG
+                  ]
+                  createBuild(
+                    buildInfo,
+                    rmsSupportedEnvironment.staging //production
+                  )
+                }
+              }
+            }
           }
-          currentBuild.description = "${env.DOCKER_TAG} & ${env.SECONDARY_DOCKER_TAG}"
         }
       }
       post {
         success {
           script {
-            if (params.RELEASE_BUILD && !params.RELEASE_DRY_RUN) {
-              Map buildInfoSecondary = [
-                "project_id": service_secondary,
-                "build_id": env.SECONDARY_DOCKER_TAG,
-                "release_id": params.RELEASE_NAME,
-                "git_tag": env.GIT_TAG
-              ]
-              createBuild(
-                buildInfoSecondary,
-                rmsSupportedEnvironment.production
-              )
-            }
+            currentBuild.description = "${env.BUILD_ID} & ${env.SECONDARY_BUILD_ID}"
           }
         }
       }
     }
-    stage('Deploys:') {
+    stage('Deploy:') {
       when {
         allOf {
           not { expression { isPr } }
           not { expression { return params.RELEASE_BUILD} }
         }
       }
       stages {
-        stage('Deploy SoQL Server PG') {
+        stage('Deploy - SoQL Server') {
           steps {
             script {
               lastStage = env.STAGE_NAME
-              // uses env.DOCKER_TAG and deploys to staging by default
-              marathonDeploy(serviceName: env.DEPLOY_PATTERN, waitTime: '60')
+              marathonDeploy(
+                serviceName: env.DEPLOY_PATTERN,
```

Comment:
It's a better practice to be explicit about parameters and put them on their own line for readability and git history clarity.